### PR TITLE
[doc] Adjusted the flink ddl document directory

### PR DESCRIPTION
**Pull request by yangjf2019**
### Purpose

Adjusted the flink ddl documentation directory.

### Documentation

**docs/content/flink/sql-ddl.md**

### CODE REVIEW

File: docs/content/flink/sql-ddl.md [hunk 1]
```diff
@@ -97,26 +99,26 @@ Also, you can create [FlinkGenericCatalog]({{< ref "flink/quick-start" >}}).
 > metastore.create.as.acid=false
 > ```
 
-#### Synchronizing Partitions into Hive Metastore
+##### Synchronizing Partitions into Hive Metastore
```
Comment (JingsongLi) [inline docs/content/flink/sql-ddl.md:102]: The hierarchy is too deep, I think it's better not to need this affiliation.

**Comment by JingsongLi** [comment]
I feel it is hard to say "easier reading and querying".

File: docs/content/flink/sql-ddl.md (hunk 1 above)
Comment (yangjf2019) [reply docs/content/flink/sql-ddl.md:102]: Well, I just keep this level the same as next.

**Comment by yangjf2019** [comment]
> I feel it is hard to say "easier reading and querying".
Yeah, I changed this describe.

File: docs/content/flink/sql-ddl.md (hunk 1 above)
Comment (yangjf2019) [reply docs/content/flink/sql-ddl.md]: I already updated it.please take a look,thank you!

**Comment by yangjf2019** [comment]
I think it is not necessary,and it can be closed.

**Pull request closed without merging**